### PR TITLE
cleaned up repo

### PR DESCRIPTION
**Pull request by longrunningprocess**
## Description

There are a number of files in the repo I don't believe are needed anymore so I decided to start cleaning those up in the hopes of speeding builds up, reducing complexity of logic that's unnecessary and helping clean up any code searches that result in hits within files that are irrelevant.

### Type of Change

- housekeeping

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the title & description of this PR which I will use as the squashed PR commit message
- [ ] I have enabled auto-merge (optional)

## How to test

- a general smoke test of the app
- ensure pictures can be uploaded

## qa.languageforge.org testing

Testers should add his/her findings to end of the PR in a comment and include screenshots, files, etc that are beneficial.

### CODE REVIEW

**Comment by longrunningprocess** [comment]
@megahirt do you have any interest in removing `transifex`?

**Comment by longrunningprocess** [comment]
@megahirt @myieye are we close to being able to remove protractor and all refs to it yet?

**Comment by longrunningprocess** [comment]
@megahirt I'm thinking `scripts` might be deletable, what are your thoughts?

image

image

image

**Comment by longrunningprocess** [comment]
@megahirt interested in removing the "site-wide-notices" capability?

**Comment by megahirt** [comment]
@longrunningprocess 

- tranifex is our UI localization solution.  Would love to transition to crowdin like everyone else however transifex is still working and it would take some effort to move
- CSV insights is a tool I wrote to get info on who is using LF and how.  Still want to use that.  
- Scripts is an environment for running PHP on the command line.  Want to keep.
-

File: upload-exclude.txt [hunk 1]
```diff
@@ -1,5 +0,0 @@
-/angular-app/**/*.js
```
Comment (megahirt) [inline upload-exclude.txt:1]: The upload-exclude.txt and include.txt files were part of a previous generation of deployment scripts that used rsync for code deployment from TeamCity.  Safe to remove.

File: LICENSE.txt [hunk 2]
```diff
@@ -20,488 +20,3 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
-Some dependencies used by this software (specifically lamejs, based on The LAME
```
Comment (megahirt) [inline LICENSE.txt:23]: great removal!

File: refreshDeps.sh [hunk 3]
```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
```
Comment (megahirt) [inline refreshDeps.sh:1]: nice removal!

File: .prettierignore [hunk 4]
```diff
@@ -38,9 +38,6 @@ next-app/**/.env.*
 
 ## Extras
 # Libraries and generated code
-src/help/template/scripts/
```
Comment (megahirt) [inline .prettierignore:41]: yes, I'm fine removing this help directory - it was shipped for a while, but we couldn't update it directly and so eventually we removed references to it.

File: src/Site/views/shared/page/scriptoutput.html.twig [hunk 5]
```diff
@@ -1,30 +0,0 @@
-{% extends app.website.base~'.html.twig' %}
```
Comment (megahirt) [inline src/Site/views/shared/page/scriptoutput.html.twig:1]: the concept I think is still good, however this file is so old that it's best to just get rid of it

File: src/Site/views/shared/page/terms_and_conditions.html.twig [hunk 6]
```diff
@@ -1,4 +1,4 @@
-{% extends app.website.base~'.html.twig' %}
```
Comment (megahirt) [inline src/Site/views/shared/page/terms_and_conditions.html.twig:1]: great find!

File: src/index.php-maintenance-message [hunk 7]
```diff
@@ -1,4 +0,0 @@
-<h2>Site Maintenance In Progress</h2>
```
Comment (megahirt) [inline src/index.php-maintenance-message:1]: In theory this file still has value to me.  I have done a number of db migrations in the past where I simply:
```
cp index.php index.php-real
cp index.php-maintenance-message index.php
[do the database migration while the site is in maintenance mode]
mv index.php-real index.php
```

If you have a different ideal on how to do this, please let me know.  This file is so small and simple that we can easily delete it now and recreate it if it's needed again.

**Review by megahirt — APPROVED** [review]
LGTM nice housekeeping!

**Pull request merged**